_posts/release/2020-12-09-accumulo-1.10.1.md

```diff
@@ -32,11 +32,16 @@ problems you encounter, as well as perform some troubleshooting steps of your
 own, in order to get the best response.
 
 ## Major Bug Fixes
-
+* Remove redundant audit message and order of authentication checks.
```

Comment:
I am not seeing this in 1.10 branch.  Did you take this from the main commit for:
`Fix unneeded audit message about denial (#1827)`

Reply:
I used the language from the PR #1831.  This text is just a place-holder until the changes are complete and the release is ready.

Reply:
OK I wasn't sure if that is going into 1.10 or not.  I asked on https://github.com/apache/accumulo/pull/1831

Reply:
1831 is unrelated. The issue you want to mention is 1828/1830.